src/middlewared/middlewared/plugins/auth_/authenticate.py

```diff
@@ -13,7 +15,7 @@ class Config:
     async def authenticate(self, username, password):
         if '@' in username:
             if (await self.middleware.call('datastore.config', 'system.settings'))['stg_ds_auth']:
-                return await self.ds_authenticate(username, password)
+                return await self.ds_authenticate(username.split('@')[0], password)
```

Comment:
Why are you removing the domain info? That tends to break PAM authentication as `pam_winbind` as it's part of the username.
```
>>> pam.authenticate("joiner", <password>)
False
>>> pam.authenticate("joiner@BILLY.GOAT", <password>)
True
```